test/Npgsql.Tests/PgPassFileTests.cs

```diff
@@ -12,31 +12,49 @@ public class PgPassFileTests
         public void ShouldParseAllEntries()
         {
             var file = new PgPassFile(_pgpassFile);
+            var fileUnique = new PgPassFile(_pgpassUniqueFile);
```

Comment:
I'm not convinced about the general test strategy - of duplicating all tests to test with two filenames.

Basically the feature you're adding is:
* A connection string parameter for a PgPassFile path
* The ability to load PgPassFile from a non-default path (the ability is already there technically, just not used).

In other words, nothing relating to the actual parsing of PgPassFile contents is affected. IMHO the only additional test that makes sense is simply one that ensures that `PgPassFile.Load(path)` works correctly for a non-null parameter.